package.json

```diff
@@ -66,7 +66,8 @@
   "scripts": {
     "lint": "yarn ts-standard",
     "fix": "yarn ts-standard --fix",
-    "test": "yarn build && cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest --runInBand",
+    "test": "jest",
```

Comment:
`jest test': would this pass locally?  As the way climb and area CRUD tests are written, they need to run sequentially within their suite, hence the --runInBand flag.

Reply:
Not really, `runInBand` continues to be a base requirement for tests to pass. I've restored it.